src/__experimental__/components/checkable-input/checkable-input.style.js

```diff
@@ -19,13 +20,17 @@ const StyledCheckableInputWrapper = styled.div`
   }) => css`
     ${FormFieldStyle} {
       display: flex;
-      flex-wrap: wrap;
+      flex-flow: row wrap;
     }
 
     ${LabelStyle} {
+      display: flex;
+      flex: 1 1 calc(100% - 28px);
+      justify-content: flex-start;
+      flex-flow: row wrap;
       text-align: ${labelAlign};
       width: auto;
-      white-space: nowrap;
+      white-space: normal;
```

Comment:
Is this white-space overwriting white-space in LabelStyle? If not, I think we can remove it because `white-space: normal` is a default value

Reply:
resolved f37ec57f3d23c58afbcb46ab4bd5cf5672be04c0